Dockerfile

```diff
@@ -8,6 +8,8 @@ ENV CONTAINER_TYPE="aplus" \
 
 COPY rootfs /
 
+RUN pip_install debugpy git+https://github.com/mher/flower#egg=flower
```

Comment:
`debugpy` is nowadays included in the service-base django layer. It is not necessary to install it again here.
https://github.com/apluslms/service-base/blob/f078293583be6d9662a7245b687f194130843986/django/Dockerfile#L6

`flower` supports the normal package version syntax nowadays and it is not necessary to install it from git.
https://github.com/apluslms/radar/blob/7647bff536690a8397d0ddc611a35281872d9621/docker/Dockerfile#L32

`flower` requires Celery. I think that when you install `flower` in a separate command here, it will already install one version of Celery at this point. When the A-plus requirements.txt is installed, the Celery version may be changed. It would make more sense to install everything at once so that the versions of the dependencies are not changed back and forth. You could combine this pip_install to the command with requirements.txt (around the line 33 below).
https://github.com/mher/flower/blob/55558ba639433bada90a6d505f81ab7eb167aff9/requirements/default.txt#LL1C1-L1C1
https://github.com/apluslms/radar/blob/7647bff536690a8397d0ddc611a35281872d9621/docker/Dockerfile#L30-L33